pkg/controller/plan/kubevirt.go

```diff
@@ -73,6 +73,8 @@ const (
 	AnnImporterPodName = "cdi.kubevirt.io/storage.import.importPodName"
 	//  Original VM name on source (value=vmOriginalName)
 	AnnOriginalName = "original-name"
+	// Openshift display name annotation (value=vmName)
+	AnnDisplayName = "openshift.io/display-name"
```

Comment:
Out of curiosity: Is this a generic annotation for any resource?

Reply:
This annotation is used in openshift (didn't find it in vanilla k8s),
It is currently an RFE for OCP-V [ CNV-59643 ]

For openshift Projects:
https://access.redhat.com/solutions/1982113
For openshift Templates:
https://docs.redhat.com/en/documentation/openshift_container_platform/3.11/html/developer_guide/dev-guide-templates#using-the-instantapp-templates

It is suggested for kubevirt VMs (on openshift clusters):
https://issues.redhat.com/browse/CNV-59643

Reply:
> Thinking if we should do something like that also for MTV resources.

:shrug: 

I added it for: https://issues.redhat.com/browse/CNV-59643

Reply:
@mnecas should we remove the "original-name" annotation ?

Reply:
hmm yeah, we are not displaying the `original-name` anywhere, right?

Reply:
> we are not displaying the original-name anywhere, right?

no, we are not using it, before this PR it showed the wrong value anyways ...

Reply:
removed the unused  "original-name" annotation